changeset-feedback/generateFeedback.ts

```diff
@@ -0,0 +1,309 @@
+/*
```

Comment:
This file is a copy/paste from backstage/backstage right? In that case I won't bother about reviewing or remark on the missing tests :)

Reply:
Yeah all copy and paste 😶‍🌫️